tests/conftest.py

```diff
@@ -38,6 +38,16 @@ def tensor_like():
     return TensorLike()
 
 
+class FloatLike:
+    def __eq__(self, other):
+        return not isinstance(other, int) and isinstance(other, float)
```

Comment:
I guess I don't have enough Python knowledge.
How the value can be an `int` and a `float` at the same time? Otherwise, I don't understand this statement.

Reply:
I thought int was a subclass of float. It is typing-wise (https://github.com/python/mypy/issues/10471#issuecomment-841206941) but not at runtime

So the non-int check could be removed

Reply:
Thanks for the info.